apps/frontend/src/components/keplerGl/factories/LayerManagerFactory.tsx

```diff
@@ -87,6 +137,23 @@ const LayerManagerFactory = (
 
     return (
       <div className="layer-manager">
+        <StyledSidePanelSection>
+          <AddDataButton onClick={showAddDataModal} isInactive={!defaultDataset} width={'auto'} />
+          <HintText>
+            <Warning stroke={themeColors.greyMedium} />
+            Un trop grand nombre de jeux de données peut altérer le projet
+          </HintText>
+
+          <p>
+            <FormattedMessage id={'layerManager.fileformat'} />
```

Comment:
Use `camelCase` for the dictionnary keys :
```suggestion
            <FormattedMessage id={'layerManager.fileFormat'} />
```